sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -559,17 +548,18 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
                 && podSpec.getNetworks().isEmpty()
                 && podSpec.getRLimits().isEmpty()
                 && secretVolumes.isEmpty()) {
+            // Nothing left to do.
             return containerInfo.build();
         }
 
-        boolean shouldAddImage = podSpec.getImage().isPresent() && addExtraParameters &&
+        boolean shouldAddImage =
```

Comment:
nittiest-pick: Is this field used anywhere else? Can it just be put directly in the if?